events/events.go

```diff
@@ -140,7 +141,10 @@ type ErrorFrame struct {
 	Message string `cborgen:"message"`
 }
 
-func (em *EventManager) AddEvent(ev *RepoStreamEvent) error {
+func (em *EventManager) AddEvent(ctx context.Context, ev *RepoStreamEvent) error {
```

Comment:
I think this will impact the labelmaker version, but I can handle that change in my code de-dupe branch. May need to check that there are no build errors after rebasing this branch.

Reply:
I tested rebasing locally, no build problem.